FlowCrypt/Core/Models/KeyDetails.swift

```diff
@@ -16,6 +16,8 @@ struct KeyDetails: Decodable {
     let isFullyEncrypted: Bool? // only if this is prv
     let ids: [KeyId]
     let created: Int
+    let lastModified: Int?
```

Comment:
lastModified should not be nil-able, all keys should have been (created or modified) at some point

Reply:
Ah! I see. Thank you for clarification.